src/components/views/Agreement.js

```diff
@@ -58,6 +59,7 @@ export default class Agreement extends React.Component {
   state = {
     showDuplicateAgreementModal: false,
     sections: {
+      supplementaryProperties: false,
```

Comment:
The convention here is that the order is the visible order of the accordion sections.